core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -138,16 +138,21 @@
   private final String host;
   private final int port;
   private final Resource<Executor> executorResource;
-  @GuardedBy("this")
+  private final long cacheTtlNanos;
+  private final SynchronizationContext syncContext;
+
+  // Must only be called from syncContext
+  private final Stopwatch stopwatch;
```

Comment:
It seems better to include this in the "must be accessed from the syncContext" section.

Reply:
Done.